master/internal/api_experiment.go

```diff
@@ -259,7 +259,7 @@ func (a *apiServer) UnarchiveExperiment(
 ) (*apiv1.UnarchiveExperimentResponse, error) {
 	id := int(req.Id)
 
-	dbExp, err := a.m.db.ExperimentByID(id)
+	dbExp, err := a.m.db.ExperimentWithoutConfigByID(id)
```

Comment:
would there be a risk of assuming config would exist on `model.Experiment` object while it isn't? a slightly safer way of doing this might be to just get the experiment state from database and doing another query to set the archive status just by requiring experiment id.

either way it get's the job done : )

Reply:
Yeah, but I figure, at least here, `ExperimentWithoutConfigByID` is an obvious enough name and all the usage is close to the call site. Idk, I wrote it that way for a moment, but didn't want two queries.

Reply:
I feel like it's so minor, if it does become and issue it can be easily changed.